app/assets/stylesheets/application.css.scss

```diff
@@ -11,6 +11,13 @@
  * file per style scope.
  *
  * 
+ *= require 'masonry/basic'
+ *= require 'masonry/centered'
+ *= require 'masonry/fluid'
+ *= require 'masonry/gutters'
+ *= require 'masonry/infinitescroll'
+ *= require 'masonry/right-to-left'
```

Comment:
Same with above. We might use those function, we are just pulling/merging, getting masonry work and will work on customisation later (after getting partials)